### ISSUE TITLE
Some interesting libraries

### ISSUE THREAD

**Issue by facine** [siwapp/siwapp-sf3#63]
Hi, I've found some great libraries of [Commerce Guys](https://github.com/commerceguys) that might be interesting to consider:

[pricing](https://github.com/commerceguys/pricing) - A PHP 5.4+ library for working with prices.
[tax](https://github.com/commerceguys/tax) - A PHP 5.5+ tax management library.
[addressing](https://github.com/commerceguys/addressing) - A PHP 5.5+ addressing library, powered by Google's dataset.

What do you think?

**Comment by ParisLiakos** [comment]
Yeah i am aware of those libraries, and was considering them back in the summer when doing the initial port, but i decided to keep the simplistic implementation so that i can get v1 out asap.
Then i definitely intend to play more with them
